### DISCUSSION TITLE
403 Forbidden after installation

### DISCUSSION THREAD

**Discussion by fullyloadedtech** [filamentphp/filament#1950] [Help] [answered]
After installing the project and log in via the Filament prompt I am redirected to /admin and it shows "403 Forbidden". I have not yet created any models or resources, is this step required to view the admin dashboard? `php artisan make:filament-resource Customer` I do not yet have a /filament folder in the root of my Laravel project.

**Comment by fullyloadedtech** [comment, marked as answer]
A user from the Discord MACscr helped me figure out the solution, for new installation it is also required to modify the User model which is not mentioned in the starter Installation documentation. Here is what is additionally required, then after doing this I was able to view the default empty Dashboard. https://filamentphp.com/docs/2.x/admin/users#authorizing-access-to-the-admin-panel

**Reply by ryangjchandler** [reply]
@fullyloadedtech The docs mention it here: https://filamentphp.com/docs/2.x/admin/installation#deploying-to-production

**Reply by fullyloadedtech** [reply]
I skipped that part because I was not testing production, however I was testing staging and not local dev. So I created this topic for the reason if anyone else hits this error they will find the solution. Here is how the default dashboard should look:
![filament-default](https://user-images.githubusercontent.com/101905909/159184678-6481a396-6579-4560-ba57-5d8121a67b63.png)

**Comment by ankheur** [comment]
I must add that these steps are also required to pass Pest tests (even in local env) like this one :
`get(PostResource::getUrl('index'))->assertOk();`

And it's also not mentionned in the documentation. If you don't you get a 403.
